### DISCUSSION TITLE
Videoscript for documenting the content and usage of: https://kg-ipclimatec-reports.wikibase.cloud/wiki/Main_Page

### DISCUSSION THREAD

**Discussion by Dansk04** [petermr/semanticClimate#44] [Show and tell]
[VideoScript_IPCC_knowledge Graph.pdf](https://github.com/petermr/semanticClimate/files/13686248/VideoScript_IPCC_knowledge.Graph.pdf)
### Hello everybody! 

Marcel (@Caliban23) and I created a draft of a videoscript for documenting the content and usage of: https://kg-ipclimatec-reports.wikibase.cloud/wiki/Main_Page

We are going to do the video soon, but before that we wanted to share this script with you.
If you have any suggestions, please let us know and we can incorporate them.

Thanks!

**Comment by petermr** [comment]
Did you include a link to the script? Suggest it could be in a Github
discussions

On Fri, Dec 15, 2023 at 2:23 PM Dansk04 ***@***.***> wrote:

> Hello everybody!
>
> Marcel ***@***.*** <https://github.com/Caliban23>) and I created a draft
> of a videoscript for documenting the content and usage of:
> https://kg-ipclimatec-reports.wikibase.cloud/wiki/Main_Page
>
> We are going to do the video soon, but before that we wanted to share this
> script with you.
> If you have any suggestions, please let us know and we can incorporate
> them.
>
> Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/semanticClimate/discussions/44>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCS7HDQ667GOCDKEO5GTYJRMPPAVCNFSM6AAAAABAWQ2G36VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZVHE3DIMRUGU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Reply by Dansk04** [reply]
The link to the documnet is on top of this discussion entry. But maybe only I see it?

**Comment by petermr** [comment]
On Mon, 18 Dec 2023, 12:43 Dansk04, ***@***.***> wrote:

> The link to the documnet is on top of this discussion entry. But maybe
> only I see it?
>
Please give link!

**Comment by petermr** [comment]
Thanks,
The test will come with the images/video :-)

BTW we prefer Markdown (*.md) files to PDF as they are semantic and editable.
